File: C++/1491. Average Salary Excluding the Minimum and Maximum Salary.cpp
class Solution {
public:
    double average(vector<int>& s) {
        double a=0.0;
        for(int i:s){
            a+=i;
        }
        a=a-*min_element(s.begin(), s.end())-*max_element(s.begin(), s.end());
        return(a/(s.size()-2));
    }
};

File: JAVA/1491. Average Salary Excluding the Minimum and Maximum Salary.java
class Solution {
    public double average(int[] s) {
        double a=s[0];
        for(int i=1;i<s.length;i++){
            a+=s[i];
        }
        Arrays.sort(s);
        a=a-s[0]-s[s.length-1];
        return(a/(s.length-2));
    }
}